src/wifi/AdafruitIO_WINC1500.h

```diff
@@ -15,15 +15,17 @@
 #ifndef ADAFRUITIO_WINC1500_H
 #define ADAFRUITIO_WINC1500_H
 
-#if !defined(ARDUINO_SAMD_MKR1000) && defined(ARDUINO_ARCH_SAMD) &&            \
```

Comment:
You should not need to modify _WINC1500.h. The MK1010 doesn't use a WINC1500 module, it has a ESP32 (u-blox 102) with nina-fw.